app/_hub/kong-inc/rate-limiting-advanced/0.31-x.md

```diff
@@ -6,15 +6,15 @@ version: 0.31-x
 
 desc: Upgrades Kong Rate Limiting with more flexibility and higher performance
 description: |
-  The Rate Limiting Advanced plugin for Kong Enterprise Edition (EE) is a
+  The Rate Limiting Advanced plugin for Kong Enterprise (EE) is a
```

Comment:
(EE) is still present

Reply:
That is addressed in https://github.com/Kong/docs.konghq.com/pull/972 This PR is only addressing Kong CE